tools/codegen/model.py

```diff
@@ -363,6 +373,28 @@ def from_yaml(ei: Dict[str, object], loc: 'Location') -> 'NativeFunction':
             cpp_no_default_args=cpp_no_default_args,
         )
 
+    @staticmethod
+    def with_dispatch_entry(f: 'NativeFunction', dispatch_key: DispatchKey, kernel: str) -> 'NativeFunction':
```

Comment:
It's a little odd this isn't a method, given that it takes a NativeFunction as argument